examples/fungible/src/contract.rs

```diff
@@ -29,19 +30,17 @@ impl WithContractAbi for FungibleTokenContract {
 }
 
 impl Contract for FungibleTokenContract {
-    type State = FungibleToken;
     type Message = Message;
     type Parameters = Parameters;
     type InstantiationArgument = InitialState;
 
-    async fn new(state: FungibleToken, runtime: ContractRuntime<Self>) -> Self {
+    async fn new(runtime: ContractRuntime<Self>) -> Self {
+        let state = FungibleToken::load(ViewStorageContext::from(runtime.key_value_store()))
+            .await
+            .expect("Failed to load state");
```

Comment:
`.expect("Loading the state should succeed")` ?

Reply:
So I was wondering about that. I think that for contract's it might be a good idea to treat the panics as error messages, because it's basically replacing the errors 🤔

Reply:
I see